### ISSUE TITLE
Getting ValueError exception

### ISSUE THREAD

**Issue by martin-dessureault** [caleb531/open-conference-url#6]
Running on macos

Was running fine until yesterday when I started to get these errors.
```
[10:37:35.824] ERROR: Open Conference URL[Script Filter] Code 1: Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Users/obfuscated/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.C9848DC8-104B-4182-A55B-18D74503C35A/ocu/list_events.py", line 115, in <module>
    main()
  File "/Users/obfuscated/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.C9848DC8-104B-4182-A55B-18D74503C35A/ocu/list_events.py", line 79, in main
    all_events = [event for event in get_events() if event.conference_url]
  File "/Users/obfuscated/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.C9848DC8-104B-4182-A55B-18D74503C35A/ocu/list_events.py", line 31, in get_events
    return [Event(event_blob) for event_blob in event_blobs]
  File "ocu/event.py", line 18, in __init__
    self.start_datetime = self.parse_start_datetime()
  File "ocu/event.py", line 41, in parse_start_datetime
    prefs.date_format, prefs.time_format))
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/_strptime.py", line 335, in _strptime
    data_string[found.end():])
ValueError: unconverted data remains: 
    location: https://us06web.zoom.us/j/obfuscated?from=addon
```

**Comment by martin-dessureault** [comment]
I deleted and reinstalled the workflow and all is good now - weird.

**Comment by caleb531** [comment]
Just noting for future reference that this was a valid bug, and has been fixed since v3.0.1 of the workflow. See #7 for the full discussion.
